test/test_collector.py

```diff
@@ -236,8 +237,10 @@ def env_fn():
     ccollector.shutdown()
 
 
-@pytest.mark.parametrize("num_env", [3, 1])
-@pytest.mark.parametrize("env_name", ["conv", "vec"])
+# @pytest.mark.parametrize("num_env", [3, 1])
```

Comment:
Any specific reason why you dropped those tests?

Reply:
This was while I was debugging. Sorry about confusion.